commands/update-check.go

```diff
@@ -0,0 +1,121 @@
+package commands
```

Comment:
I'd prefer to leave it where it is and make it a package private function. I don't see this function being used outside of this package, so I don't think it should be put in `util`.

Reply:
Adding the `UpdateCheck()` function to the `version` command requires that `UpdateCheck()` be exposed since it must be added before `cli.Run()` is called.

`UpdateCheck()` can be made package private if the check is removed from `version` or an implementaion of `version` is added to the `command`.

Reply:
Let's add a function in the command package `GetVersionCommand` or `GetVersionPrinter` which returns a `func (c *cli.Context)`. That way, the version command logic is in the command package and can use `UpdateCheck` as a package private function.

Reply:
I put `GetVersionPrinter()` into `update-check.go` and unexposed `updateCheck()`.